MachineLearning_Engine/Convert/ToList.cs

```diff
@@ -34,20 +36,30 @@ public static partial class Convert
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Convert a Tensor to a list of data.")]
```

Comment:
A list of what type of data? Will it always be double by default or could we later convert to a list of string? If always double then don't worry about changing.

Reply:
To be honest I'm not sure. Currently this method is only capable to handle double as Eduardo noted and will only be used for regression methods which always return double. He did put a TODO here to handle other types but I'm not sure where the functionality will be used.

Reply:
Ok fair enough, let's leave as is then :smile: